system-tests/docker-compose.yml

```diff
@@ -104,15 +104,57 @@ services:
     working_dir: /
     environment:
       API_KEY: ApiKeyDefaultValue
-      EDC_HOST: provider
-      ASSETS_STORAGE_ACCOUNT: providerassets
+      EDC_HOST: company1
+      ASSETS_STORAGE_ACCOUNT: company1assets
       PARTICIPANT_ID: company1
     depends_on:
```

Comment:
Does it really need all connectors to be ready? Or just company1? Same for the others below.

Reply:
Updated to use only one newman container.